docs/topics/Permissions.md

```diff
@@ -35,7 +35,7 @@ Below is a table of all current permissions, their integer values in hexadecimal
 | MANAGE_CHANNELS \*    | `0x0000000010` `(1 << 4)`  | Allows management and editing of channels                                                                                          | T, V, S      |
 | MANAGE_GUILD \*       | `0x0000000020` `(1 << 5)`  | Allows management and editing of the guild                                                                                         |              |
 | ADD_REACTIONS         | `0x0000000040` `(1 << 6)`  | Allows for the addition of reactions to messages                                                                                   | T            |
-| VIEW_AUDIT_LOG        | `0x0000000080` `(1 << 6)`  | Allows for viewing of audit logs                                                                                                   |              |
+| VIEW_AUDIT_LOG        | `0x0000000080` `(1 << 7)`  | Allows for viewing of audit logs                                                                                                   |              |
```

Comment:
the value is still the same though, looks like the ones below are also just continued based on this one

Reply:
The value is correct with `1<<7 == 0x80` and `1<<6 == 0x40`